sdk/python/kfp/_client.py

```diff
@@ -32,6 +32,7 @@
 from kfp.compiler._k8s_helper import sanitize_k8s_name
 
 from kfp._auth import get_auth_token, get_gcp_access_token
+from kfp._credentials import ServiceAccountTokenVolumeCredentials
```

Comment:
Import modules instead: https://google.github.io/styleguide/pyguide.html#22-imports

Reply:
Because each credential can be optional, I'd suggest import this dynamically only when we know what we want to use.

Reply:
Done!